templates/_includes/overview-footer.html

```diff
@@ -1,20 +1,20 @@
 <div class="overview-footer">
   <div class="footer-content">
     <div class="hxRow">
-      <div class="hxCol">
+      <div class="hxCol footer-content">
```

Comment:
For the Helix grid, we should have a separate container for styling like `.footer-content`. See https://rackerlabs.github.io/helix-ui/components/grid/#styling

Reply:
I see `.footer-content` used two levels up, as well. Is this desired?

Reply:
good point; I need to remember to expand the review :joy: It shouldn't be desired, no.

Reply:
@technabors -- I see the footer-content class repeated in the footer. I think that second one can be cut, no?

Reply:
Removed the extra use of the class. Unsure what I was even doing here. Post change all footer content updates are still styled correctly.